cpg_infra/config/deserializabledataclass.py

```diff
@@ -168,6 +168,12 @@ def try_parse_value_as_type(value: Any, dtype: Any) -> Any:
             )
         return tuple(try_parse_value_as_type(v, t) for v, t in zip(value, tuple_types))
 
+    if str(getattr(dtype, '__origin__', None)) == 'typing.Literal':
```

Comment:
Is there a reason for not doing this as `if get_origin(dtype) is Literal`?

Reply:
Is because I'm dumb valid?

Reply:
No, but I'd accept "temporarily forgot about that API" 😄